crowdsec-docs/docs/data_sources/loki.md

```diff
@@ -0,0 +1,104 @@
+---
+id: loki
+title: Loki
+---
+
+This module allows the `Security Engine` to acquire logs from loki query.
+
+## Configuration example
+
+This will allow to read logs from loki, using the query `{job="varlogs"}`.
+```yaml
+source: loki
+log_level: info
+url: http://localhost:3100/
+limit: 1000
+query: |
+  {job="varlogs"}
+auth:
```

Comment:
Does acquisition support variables?

Reply:
wdym ?

Reply:
You might not to store authentication credentials as plain text